src/main/java/frc/team2412/robot/commands/limelight/GetWithinDistanceCommand.java

```diff
@@ -0,0 +1,43 @@
+/*package frc.team2412.robot.commands.limelight;
+
+
+public class GetWithinDistanceCommand extends Command {
+
```

Comment:
Consider uncommenting this, and just not using the command yet.

Reply:
The whole GetWithinDistanceCommand.java file is redundant and has been removed